### PR TITLE
chore(core): export schema definition

### PR DESCRIPTION
**Pull request by mallachari**
Needed for https://github.com/stoplightio/platform-internal/issues/9297

**Checklist**

- [ ] Tests added / updated
- [ ] Docs added / updated

**Does this PR introduce a breaking change?**

- [ ] Yes
- [x] No

### CODE REVIEW

File: packages/core/src/ruleset/index.ts [hunk 1]
```diff
@@ -1,6 +1,6 @@
 export { assertValidRuleset, RulesetValidationError } from './validation';
 export { getDiagnosticSeverity } from './utils';
-export { createRulesetFunction } from './rulesetFunction';
+export { createRulesetFunction, SchemaDefinition } from './rulesetFunction';
```
Comment (P0lip) [inline packages/core/src/ruleset/index.ts]:
```suggestion
export { createRulesetFunction, SchemaDefinition as RulesetFunctionSchemaDefinition } from './rulesetFunction';
```

**Review by P0lip — COMMENTED** [review]
:+1:
Let's just re-export it under a different name to avoid confusion.

**Comment by mallachari** [comment]
Actually the changes you made yesterday for studio `RuleFunctionOptions` solve the problem so we might not need that type for now. Unless you think that it may be better to use it instead or might be useful in future.

**Review by P0lip — APPROVED** [review]

**Comment by P0lip** [comment]
> Actually the changes you made yesterday for studio RuleFunctionOptions solve the problem so we might not need that type for now. Unless you think that it may be better to use it instead or might be useful in future.

Yeah, was about to say we're probably good in terms of Studio, but it won't hurt to have that exported.

**Pull request merged**